dbt/models/default/schema/default.vw_card_res_char.yml

```diff
@@ -112,7 +112,7 @@ models:
     data_tests:
       - row_count:
           name: default_vw_card_res_char_row_count
-          above: 28450039 # as of 2024-12-06
+          above: 28450000 # as of 2024-12-23
```

Comment:
Note that this is not the exact count of records (28,450,037), but I wanted to set a slightly lower threshold to give us more time before we need to adjust this threshold again.